wikipedia-dark.user.css

```diff
@@ -735,9 +768,35 @@ regexp("https?:\/\/wiki\.(archlinux|mozilla)\.(org|jp)\/.*$") {
   tr > td[style="background: #CFE8FF;"], tr > td[style="background: #EED8AE;"],
   tr > td[style="background: #FFF8DC;"], tr > td[style="background: #B9D3FF;"],
   tr > td[style="background: #6495ED;"], tr > td[style="background: #607CD2;"],
-  tr > td[style="background: #828BD9;"], tr > td[style="background: #F0F8FF;"] {
+  tr > td[style="background: #828BD9;"], tr > td[style="background: #F0F8FF;"],
+  .infobox tr[style*="background:#F6E5A8;"] td,
+  .infobox tr[style*="background:#F6E5A8;"] th,
+  td[style*="background:#AED0F0"],
+  #mw-content-text .infobox-above[style*="background-color: lightblue"],
+  #mwAg .infobox-above[style*="background-color: lightblue"],
+  #mw-content-text .infobox-above[style*="background: #CCCCFF;"],
+  #mwAg .infobox-above[style*="background: #CCCCFF;"],
+  #mwAg .infobox-above[style*="background-color: lightblue"],
+  #mw-content-text .infobox-above[style*="background-color: #e7dcc3;"],
+  #mwAg .infobox-above[style*="background-color: #e7dcc3;"],
+  #mw-content-text .infobox-above[style*="background-color:  #c9ffd9;"],
+  #mw-content-text .infobox-above[style*="background:#F6DA9F"],
+  #mw-content-text .infobox tr[style*="background:#F8C9B0;"] td,
```

Comment:
Generally, the headers on infoboxes should be light text on a dark background, to match the inverted style of the rest of the page. This would mean finding a darker version of the light background that works well and has a high enough contrast with the text.

Of course, this is an uphill battle, and we would end up adding dozens more colors to our palette (which isn't neccessarily a bad thing).  If there is an existing color defined in the style that would work as a replacement, we should use that instead. So either way works, I suppose.